src/components/HeartFavorite/HeartFavorite.style.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const HeartButton = styled.button`
+  background: none;
+  border: none;
+`;
```

Comment:
You should create a new Button variant instead, check how `DropboxButton` is created.

Reply:
thats the entire point of the styled components @ImAriel2D